akka-stream-testkit/src/test/scala/akka/stream/testkit/StreamTestKitSpec.scala

```diff
@@ -121,13 +122,17 @@ class StreamTestKitSpec extends AkkaSpec {
     "#expectNextWithTimeoutPF should fail after timeout when element delayed" in {
       intercept[AssertionError] {
         val timeout = 100.millis
-        val overTimeout = timeout + 50.millis
+        // Initial delay is longer than the timeout so an exception will be thrown.
+        // It also needs to be dilated since the testkit will dilate the timeout
+        // accordingly to `-Dakka.test.timefactor` value.
+        val initialDelay = (timeout * 2).dilated
```

Comment:
This is now also doubling the timeout so that the tests will have a larger difference between values.

Reply:
yes, there is no reason to make this too short